prow/plugins/label/label.go

```diff
@@ -205,6 +219,24 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 		}
 	}
 
+	for _, statusMatch := range statusMatches {
+		status := strings.TrimSpace(statusMatch[1])
+		sLabel, validStatus := statusMap[status]
+		if validStatus {
+			// true is for testing purposes only, need to figure out way to  mock for testing
+			if true || hasStatusPrivilege(ae.login) {
```

Comment:
This seems fine to me, aside from the `true || ` which I understand is only temporary.